fog/view/server/src/router_request_handler.rs

```diff
@@ -39,6 +39,7 @@ where
 {
     while let Some(request) = requests.try_next().await? {
         let _timer = SVC_COUNTERS.req_impl(&method_name);
+        CONNECTED_CLIENTS.inc();
```

Comment:
Why is this inside the loop that is handling requests? Wouldn't this mean we'll count a stream that has multiple requests as multiple clients?